documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,266 @@ You may obtain a copy of the License at
-many Events can be in a Stream or how many total bytes are stored in a Stream.
-Pravega’s design horizontally scales from few machines to a whole datacenter’s capacity.
-
-To deal with a potentially large amount of data within a Stream, Pravega Streams
-are divided into Stream Segments.  A Stream Segment is a shard, or partition of
-the data within a Stream.  We go into a lot more detail on Stream Segments a bit
-later in this document.  Stream Segments are an important concept, but we need
-to introduce a few more things before we can dive into Stream Segments.
-
-Applications, such as a Java program reading from an IoT sensor, write data to
-the tail (front) of the Stream.  Applications, such as a [Flink](https://flink.apache.org) analytics job,
-can read from any point in the Stream. Lots of applications can read and write
-the same Stream in parallel.  Elastic, scalable support for a large volume of
-Streams, data and applications is at the heart of Pravega's design.  We will
-get into more details about how applications read and write Streams a bit later
-in this document when we detail Readers and Writers.
+Pravega organizes data into **Streams**.  A stream is a durable, elastic, append-only, unbounded sequence of bytes having good performance and strong consistency.  A Pravega stream is
+similar to but more flexible than a "topic" in popular message oriented middleware such as [RabbitMQ](https://www.rabbitmq.com/) or [Apache Kafka](https://kafka.apache.org/).
+
+Pravega streams are based on an append-only log data structure. By using the
+append-only logs, Pravega rapidly ingest data into durable storage. It supports a large variety of application use cases as follows:
+
+- Supports stream processing by using frameworks like [Flink](https://flink.apache.org).
+- Publish/subscribe messaging.
+- NoSQL databases like Time Series.
+- Database (TSDB).
+- Workflow engines.
+- Event-oriented applications, etc., 
+
+In Pravega, during the creation of the stream the developers assigns a meaningful
+name to the stream like "IoTSensorData" or "WebApplicationLog20170330" to provide
+more information on the type of data stored in the stream. The Pravega stream names are organized
+within a **Scope**.  A scope is a string containing meaningful informations like "FactoryMachines" or "HRWebsitelogs" to aid in better understanding for the developers. A scope acts as a
+namespace for stream names, all stream names are unique within a scope.
+
+A stream is uniquely identified by its combination of the **stream name**
+and **scope**. Scope can be used to classify names by tenant (in a multi tenant
+environment) or by department in an organization or by geographic location or any
+other categorization desired by the developer.
+
+A stream is unbounded in size. Pravega does not impose any limits on the occurrence of number of events in the stream or on the number of total bytes that are stored in a stream.
```

Comment:
Change this, is not clear: ` the occurrence of number of events`.

Reply:
Accepted Suggestion